ssh/tunnel.py

```diff
@@ -16,8 +16,6 @@
 
 from pkgpanda.util import write_string
 
-LOGGING_FORMAT = '[%(asctime)s|%(name)s|%(levelname)s]: %(message)s'
```

Comment:
If we have 5 invokers, we  don't want to add this backend in everyone of them. It might helpful to just provide the "logging infrastructure" that new modules or clients can just use.


I think, my bias is coming from seeing this 'pattern' in many frameworks
For e.g sqlalchemy goes a step further and defines logging configuration classes:

https://github.com/zzzeek/sqlalchemy/blob/64e72b58fd74db0c5c87407c824d08ad699e5e8e/lib/sqlalchemy/log.py#L51-L51


* This PR is an improvement, so +1 to that. Keeping the common logger in mind might be a good idea to implement when we feel the need for it.